src/jobs/TrainDoc2Vec.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import string
+from typing import List
+
+import nltk
+from gensim.models import Doc2Vec
+from gensim.models.doc2vec import TaggedDocument
+from nltk.corpus import stopwords
+
+from jobs.AbstractJob import AbstractJob
```

Comment:
FYI there's a canonical order for imports in the PEP8 style guide. Pycharm can do it for you with (I think) ctrl+alt+o

Reply:
This is what pycharm did for me. It didn't change when I optimized the imports...

Reply:
Hmm ok, it just looked weird to me 💯